frontend/package.json

```diff
@@ -27,8 +27,10 @@
         "react-dom": "17.0.2",
         "react-icons": "^4.3.1",
         "react-responsive-carousel": "^3.2.23",
+        "react-router-dom": "^6.2.2",
         "recharts": "^2.1.9",
         "sass": "^1.49.0",
+        "socket.io-client": "^4.1.3",
```

Comment:
why is socket.io-client a required package in the backend?

Reply:
Just for the T E S T S cause to test the socket server and its functionalities, we have to use both the socket client and the socket server

Reply:
Awesome, thanks for clarifying! I love tests

Reply:
No problem ! I'm glad you're enjoying tests, I haven't done the frontend testing portion yet, so feel free to do it !